.eslintrc.json

```diff
@@ -31,7 +31,10 @@
     "no-unused-vars": "off",
     "@typescript-eslint/no-unused-vars": "warn",
     "prettier/prettier": [
-      "error"
+      "error",
+      {
+        "trailingComma": "es5"
```

Comment:
Why are we changing this?

Reply:
Linting used to fail but it's not anymore. Removing it